.github/workflows/deploy.yml

```
@@ -0,0 +1,77 @@
+          host: ${{ secrets.VPS_HOST }}
+          username: ${{ secrets.VPS_USERNAME }}
+          key: ${{ secrets.VPS_SSH_KEY }}
+          script: |
+            cd /path/to/your/app
+            # Copy the .env file to the server
+            cat > .env << 'EOL'
+            ${{ secrets.ENV_FILE_CONTENT }}
+            EOL
+            docker pull ${{ env.DOCKER_IMAGE }}:${{ env.DOCKER_TAG }}
+            docker-compose down
+            docker-compose up -d 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)